src/app.rs

```diff
@@ -617,6 +624,19 @@ impl Application for App {
                     return self.update_config();
                 }
             }
+            Message::ContextCloseAny => {
+                let entity = self.tab_model.active();
+
+                // Close menus and context panes in order per message
```

Comment:
Rather than do a Message, I would recommend to move this code to the on_escape function

Reply:
Alright. I refactored it.